DEPS

```diff
@@ -42,6 +42,7 @@ deps = {
   },
   "third_party/playlist_component/src": "https://github.com/brave/playlist-component.git@5434730bf7342f1ba5c057f1640882bb38604a85",
   "third_party/constellation/crate": "https://github.com/brave/constellation.git@b6b8396abd98cc87a187e051c32a291c9faa43f7",
+  "third_party/macholib": "https://github.com/ronaldoussoren/macholib.git@36a6777ccd0891c5d1b44ba885573d7c90740015",
```

Comment:
why do we need this?

Reply:
It's used in signature_generator_python3.py, which can be found in brave/devops.

Reply:
(it decompiles the Mac binaries somehow for the sig file generation)

Reply:
if this is macos only, can we add a condition for it?